recipe/conda_build_config.yaml

```diff
@@ -478,7 +478,7 @@ libunwind:
 libwebp:
   - 1.0.0
 libxml2:
-  - 2.9
+  - 2.9.*
```

Comment:
AFAIK, there's no difference

Reply:
Then I'm confused why https://github.com/conda-forge/r-base-feedstock/pull/97 is pulling in `libxml2=2.9` and not `libxml2=2.9.9`

Reply:
Yes, I just failed to correctly interpret the conda error message, closing.